ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-ci-pipeline.yml

```diff
@@ -226,6 +227,29 @@ spec:
           workspace: pipeline
           subPath: src
 
+    - name: get-cert-project-related-data
```

Comment:
Is it actually okay to just add this task to the CI pipeline? It doesn't just get cert project data, it also performs validations, and if those validations are not met, the task fails. This means that we're introducing requirements to the CI pipeline that weren't there before.

Reply:
decided not to edit CI pipeline